### PR TITLE
simplify proto generate command

### PR DESCRIPTION
**Pull request by yaoshengzhe**
Use single line shell command to generate proto file instead of
hard coding command for each proto.

Steps:
  1. list all proto files.
  2. remove 'proto/' prefix and '.proto' suffix.
  3. run protoc for each proto and put in go/vt/proto/${proto_file_name}

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -184,26 +184,11 @@ bson:
...
-	cd go/vt/proto/vtworkerdata && $$VTROOT/dist/protobuf/bin/protoc -I../../../../proto ../../../../proto/vtworkerdata.proto --go_out=plugins=grpc:.
-	cd go/vt/proto/vtworkerservice && $$VTROOT/dist/protobuf/bin/protoc -I../../../../proto ../../../../proto/vtworkerservice.proto --go_out=plugins=grpc:.
-	cd go/vt/proto/tabletmanagerdata && $$VTROOT/dist/protobuf/bin/protoc -I../../../../proto ../../../../proto/tabletmanagerdata.proto --go_out=plugins=grpc:.
-	cd go/vt/proto/tabletmanagerservice && $$VTROOT/dist/protobuf/bin/protoc -I../../../../proto ../../../../proto/tabletmanagerservice.proto --go_out=plugins=grpc:.
-	cd go/vt/proto/automation && $$VTROOT/dist/protobuf/bin/protoc -I../../../../proto ../../../../proto/automation.proto --go_out=plugins=grpc:.
-	cd go/vt/proto/automationservice && $$VTROOT/dist/protobuf/bin/protoc -I../../../../proto ../../../../proto/automationservice.proto --go_out=plugins=grpc:.
+	ls proto/*.proto | sed 's/^proto\///' | sed 's/.proto//' | xargs -I{} $$VTROOT/dist/protobuf/bin/protoc -Iproto proto/{}.proto --go_out=plugins=grpc:go/vt/proto/{}
```
Comment (alainjobart) [inline Makefile]: I think ls's output cannot be trusted, so you probably should use 'find', like 'find proto/ -name *.proto -print'

**Comment by alainjobart** [comment]
LGTM after comment is addressed

File: Makefile (hunk 1 above)
Comment (michael-berlin) [inline Makefile]: "." needs to be escaped in sed expression:

sed 's/\.proto//'

**Comment by yaoshengzhe** [comment]
1. replace 'ls' with 'find' command.
2. escape '.' in sed.

**Comment by alainjobart** [comment]
LGTM

**Pull request merged**